.github/actions/container-run/action.yml

```diff
@@ -3,31 +3,22 @@ description: Run a docker container
 
 inputs:
   registry:
-    description: Server address ofthe  Docker registry.
+    description: Container registry.
     required: false
-    default: osconfig.azurecr.io
-  username:
-    description: Username used to login to the Docker registry.
-    required: true
-  password:
-    description: Password or personal access token used to login to the Docker registry.
-    required: true
-  os:
-    description: The container OS.
-    required: true
-  arch:
-    description: The container architecture.
+    default: ""
+  container:
+    description: The name of the container.
     required: true
   platform:
-    description: The multiarch platform.
+    description: The container architecture.
     required: true
   mount:
     description: The volume to mount.
     required: true
   tag:
     description: The container tag.
     required: false
-    default: "289651"
+    default: latest
```

Comment:
Should we always use the latest? Will the latest always be fully vetted?

Reply:
`latest` will be fully vetted by the container build workflow. New container images are tested at PR time (gating the new image from being used) before they are merged to main (when they will then be pushed to our container registries).